c1.js

```diff
@@ -0,0 +1,26 @@
+"use strict"
+
+// solution 1
+
+function printAllPrimes(){
+    var primes = []
```

Comment:
Their main difference is their access scope.`var` have function scope, `let` have block scope (between `{}`) and `const` is similar to let but it can not be re assigned. 
In this case, I think there is no different between var, let and const and the code will work the same.
Do you recommend to change it to const or let ?

Reply:
I recommend always using `const` or `let` instead of `var` to create a healthy habit of using it as the default approach.